### PR TITLE
Pull stellar-cli docs from latest released version.

### LINKED ISSUE #1067: CLI docs is pulling from main but should pull from release branch

### PR DESCRIPTION
**Pull request by fnando**
Close #1067

### CODE REVIEW

File: scripts/stellar_cli.mjs [hunk 1]
```diff
@@ -1,23 +1,26 @@
 import fs from "fs-extra";
-import path from 'path';
```
Comment (fnando) [inline scripts/stellar_cli.mjs:2]: all quotes were changed based on `eslint` formatting.

File: scripts/stellar_cli.mjs [hunk 2]
```diff
@@ -1,23 +1,26 @@
...
 }
 
 // Perform a shallow clone of the repository
-console.log('Cloning repository...');
-execSync(`git clone --depth 1 ${repoUrl} ${localRepoPath}`);
+console.log("Cloning repository...");
+execSync(`git clone ${repoUrl} ${localRepoPath}`);
+execSync(
+  `cd ${localRepoPath} && git checkout --quiet $(git tag | grep -v -E 'rc|preview' | tail -n1)`,
+);
```
Comment (fnando) [inline scripts/stellar_cli.mjs:19]: The crux of the change: pull the latest tag that's not rc or preview.

File: docs/data/horizon/api-reference/structure/consistency.mdx [hunk 3]
```diff
@@ -3,7 +3,4 @@ title: Consistency
...
-The value of the `Latest-Ledger` HTTP header is the sequence number of the latest ledger known to Horizon at the time the request was processed.
-Horizon will guarantee that all the data included in the response is consistent with that ledger. This mechanism prevents race conditions where
-a request is processed at the boundary of two ledgers and ensures that the response is consistent with the ledger included in the `Latest-Ledger` HTTP header. 
+For endpoints which serve data which can change from ledger to ledger (for example an account balance), Horizon includes a `Latest-Ledger` HTTP header in its response. The value of the `Latest-Ledger` HTTP header is the sequence number of the latest ledger known to Horizon at the time the request was processed. Horizon will guarantee that all the data included in the response is consistent with that ledger. This mechanism prevents race conditions where a request is processed at the boundary of two ledgers and ensures that the response is consistent with the ledger included in the `Latest-Ledger` HTTP header.
```
Comment (fnando) [inline docs/data/horizon/api-reference/structure/consistency.mdx:6]: `yarn run check:mdx` was complaining about this file.

**Comment by briwylde08** [comment]
Lgtm! But will defer final review and approval to @ElliotFriend.

File: .gitignore [hunk 4]
```diff
@@ -23,7 +23,10 @@ yarn-error.log*
 
 *.info.mdx
 
+.tool-versions
+
```
[the remark this answers is not in the document]
Comment (fnando) [reply .gitignore:27]: This is [ASDF](https://asdf-vm.com)'s version file. It manages runtimes (just like nvm), but it's language agnostic, so you can control python, ruby, etc with just one CLI.

File: .gitignore [hunk 5]
```diff
@@ -23,7 +23,10 @@ yarn-error.log*
...
 # non-production openrpc.json files
 /openrpc/*openrpc.json
 
 stellar-cli-repo
+stellar-cli
```
[the remark this answers is not in the document]
Comment (fnando) [reply .gitignore:32]: Oh, I was messing around. This shouldn't be there. Let me remove it.

File: .gitignore (hunk 4 above)
[the remark this answers is not in the document]
Comment (ElliotFriend) [reply .gitignore:27]: oooohh... TIL. I'm gonna have to check that out! Thanks!

**Review by ElliotFriend — APPROVED** [review]
lgtm! Thanks @fnando

**Pull request merged**